src/app/recipe/new/page.tsx

```diff
@@ -0,0 +1,87 @@
+import { mockDataNewRecipe } from "@/mock/NewRecipe";
+
+import { CloseButton, Edit, Plus } from "@/components/icons";
+import { ImageCarousel, ImageComponent } from "@/components/image";
+import { AddButton } from "@/app/recipe/new/_components/AddButton";
+import { IngredientListItem } from "@/app/recipe/new/_components/IngredientListItem";
+import { RecipeStepItem } from "@/app/recipe/new/_components/RecipeStepItem";
+
+const Page = () => {
+  const titleClass = "mb-1 ml-4 font-bold text-black";
+  return (
+    <div>
+      <div className="flex items-center justify-between border-b border-lightGray px-3 py-4">
+        <CloseButton />
+        <div className="flex items-center gap-4 font-bold">
+          <button className="text-gray">下書き</button>
+          <button className="text-tomato">作成する</button>
+        </div>
+      </div>
```

Comment:
コンポーネントの解除ありがとうございます！
headerタグにするとマークアップの意味合い的にもあって、他の開発者もどこに何があるか分かりやすくなるので変更お願いします。

```suggestion
      <header className="flex items-center justify-between border-b border-lightGray px-3 py-4">
        <CloseButton />
        <div className="flex items-center gap-4 font-bold">
          <button className="text-gray">下書き</button>
          <button className="text-tomato">作成する</button>
        </div>
      </header>
```